monai/apps/datasets.py

```diff
@@ -241,11 +245,28 @@ def __init__(
                 f"Cannot find dataset directory: {dataset_dir}, please use download=True to download it."
             )
         data = self._generate_data_list(dataset_dir)
+        if property_keys is not None:
+            self._properties = load_decathlon_properties(os.path.join(dataset_dir, "dataset.json"), property_keys)
+        else:
+            self._properties = None
         super().__init__(data, transform, cache_num=cache_num, cache_rate=cache_rate, num_workers=num_workers)
 
     def randomize(self, data: Optional[Any] = None) -> None:
         self.rann = self.R.random()
 
+    def get_properties(self, keys: Optional[Union[Sequence[str], str]] = None):
+        """
+        Get the loaded properties of dataset with specified keys.
+        If no keys specified, return all the loaded properties.
+
+        """
+        if keys is None:
+            return self._properties
+        elif self._properties is not None:
+            return {key: self._properties[key] for key in ensure_tuple(keys)}
```

Comment:
what if `key not in self.properties`?

Reply:
I think it will raise an exception by dict object directly?
Thanks.